CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.1.0]
+### Added
+- Initial release
+- Deploy example site to Netlify [#6](https://github.com/azavea/react-showtime/pull/6)
+- Create workflow to publish to npm [#7](https://github.com/azavea/react-showtime/pull/7)
+
+[Unreleased]: https://github.com/azavea/react-showtime/compare/0.1.0...HEAD
+[0.1.0]: https://github.com/azavea/react-showtime/releases/tag/0.1.0
```

Comment:
This tag doesn't exist yet but it will be created right after this PR is merged.